workspace.html

```diff
@@ -118,7 +152,6 @@ <h3>Private Office</h3>
             <h6>Space to suit your team</h6>
             <h6>Up To 20 Conference Room Hours Per Month</h6>
             <div class="abs-btn long-text"><a class="lgx-btn" href="#" data-toggle="modal" data-target="#member-form-modal"><span>Starting at</span> $400/Month</a></div>
-          </div>
```

Comment:
@sgha229 Looks like a closing div tag was removed. I was able to find this easily with my text editor. I'd really encourage to switch your environment to some text editor instead of vim.